### PR TITLE
DocFixit: Add more troubleshooting details

### PR DESCRIPTION
**Pull request by sreecha**

### CODE REVIEW

File: src/python/grpcio/README.rst [hunk 1]
```diff
@@ -50,10 +50,43 @@ Troubleshooting
 Help, I ...
 
 * **... see a** :code:`pkg_resources.VersionConflict` **when I try to install
-  grpc!**
+  grpc:**
```
Comment (soltanmm) [inline src/python/grpcio/README.rst]: nit: a colon implies a follow up phrase by the same speaker. Here, the voice is that of the user, but what follows is our voice. If the format of the 'trouble' wasn't in the first-person form from the perspective of the user, I might think a colon is fine. As it stands, though, this should be something that maintains consistency with our empathetic use of the user's voice.

Comment (sreecha) [inline src/python/grpcio/README.rst]: Good point. I made the change.

**Comment by soltanmm** [comment]
It seems like the overall style of having a brief empathetic statement may have been a slight mistake on my part. I'll make a pass later to change up the troubleshooting format and address any other nits I'm going to take a pass on for now (and then manually update the PyPI page with those changes). In the meanwhile, lgtm.

**Comment by sreecha** [comment]
Sounds good Masood. Perhaps you can fix it as a part of https://github.com/grpc/grpc/issues/5831

**Comment by jtattermusch** [comment]
Merging without green tests as this is just docs.

**Pull request merged**